ReactiveCocoaTests/Swift/ObjectiveCBridgingSpec.swift

```diff
@@ -14,27 +14,97 @@ import ReactiveCocoa
 class ObjectiveCBridgingSpec: QuickSpec {
 	override func spec() {
 		describe("RACSignal.asSignalProducer") {
-			pending("should subscribe once per start()") {
+			it("should subscribe once per start()") {
+				var subscriptions = 0
+
+				let racSignal = RACSignal.createSignal { subscriber in
+					subscriber.sendNext(subscriptions++)
+					subscriber.sendCompleted()
+
+					return nil
+				}
+
+				let producer = racSignal.asSignalProducer() |> map { $0 as Int }
+
+				expect((producer |> single)?.value).to(equal(0))
+				expect((producer |> single)?.value).to(equal(1))
+				expect((producer |> single)?.value).to(equal(2))
 			}
 
-			pending("should automatically replace nil NSErrors") {
+			it("should automatically replace nil NSErrors") {
+				let racSignal = RACSignal.error(nil)
+				let producer = racSignal.asSignalProducer()
+				let result = producer |> last
+
+				expect(result?.error?.localizedDescription).to(equal("Nil RACSignal error"))
 			}
 		}
 
 		describe("asRACSignal") {
 			describe("on a Signal") {
-				pending("should forward events") {
+				it("should forward events") {
+					let (signal, sink) = Signal<NSNumber, NoError>.pipe()
+					let racSignal = asRACSignal(signal)
+
+					var lastValue: NSNumber?
+					var didComplete = false
+
+					racSignal.subscribeNext({ number in
+						lastValue = number as? NSNumber
+					},
+					completed: {
+						didComplete = true
+					})
+
+					expect(lastValue).to(beNil())
+
+					for number in [1, 2, 3] {
+						sendNext(sink, number)
+						expect(lastValue).to(equal(number))
+					}
+
+					expect(didComplete).to(beFalse())
+					sendCompleted(sink)
+					expect(didComplete).to(beTrue())
 				}
 
-				pending("should convert errors to NSError") {
+				it("should convert errors to NSError") {
+					let (signal, sink) = Signal<AnyObject, TestError>.pipe()
+					let racSignal = asRACSignal(signal)
+
+					let expectedError: TestError = .Error2
+					var error: NSError?
+
+					racSignal.subscribeError { error = $0; return }
```

Comment:
:-1: on semicolons anywhere. :stuck_out_tongue_winking_eye: 

Can you please split this over multiple lines?